types/stripe/index.d.ts

```diff
@@ -5912,6 +5961,62 @@ declare namespace Stripe {
         }
     }
 
+    namespace taxRates {
+        /**
+         * Tax rates can be applied to invoices and subscriptions to collect tax.
+         */
+        interface ITaxRate extends IResourceObject {
+             /**
+              * String representing the object’s type. Objects of the same type share the same value.
+              */
+             object: "tax_rate";
```

Comment:
Did this pass the linter?  I'm surprised.  I thought this project had done a big migration to single quotes and the linter would enforce that.

Reply:
Thank you for the review @pushplay 
If it did get caught by the linter then I must have missed it, my apologies.
I've updated it and this time there are definitely no linting issues :)
```
husky > pre-commit (node v10.16.0)
✔  precise-commits: 1 modified file(s) found
✔  [1/1] Processing file: types/stripe/index.d.ts
ℹ        --> No formatting changes required in: types/stripe/index.d.ts
✔  Formatting complete 🎉
```